README.md

```diff
@@ -502,6 +502,8 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Jersey](https://jersey.java.net/) - JAX-RS reference implementation.
 * [RESTEasy](http://resteasy.jboss.org/) - Fully certified and portable implementation of the JAX-RS specification.
 * [RestExpress](https://github.com/RestExpress/RestExpress) - Thin wrapper on the JBOSS Netty HTTP stack to provide scaling and performance.
+* [Restlet](http://restlet.com/projects/restlet-framework/)) - Restlet Framework helps Java developers build better web APIs that follow the REST architecture style.
```

Comment:
Unfortunately, you have a double `)`. Description is too generic and doesn't say anything.